crates/cargo-util/src/du.rs

```diff
@@ -74,5 +94,9 @@ fn du_inner(path: &Path, patterns: &[&str]) -> Result<u64> {
         return Err(e);
     }
 
-    Ok(total.load(Ordering::Relaxed))
+    #[cfg(target_has_atomic = "64")]
+    return Ok(total.load(Ordering::Relaxed));
```

Comment:
I needed to use a return here so it would compile. Rust doesn't know that the two `cfg` blocks are mutually exclusive.